<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FvUpdateWindow</class>
 <widget class="QDialog" name="FvUpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Software Update</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../vmisc/share/resources/icon.qrc">
    <normaloff>:/icon/64x64/icon64x64.png</normaloff>:/icon/64x64/icon64x64.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="newVersionIsAvailableLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>A new version of %1 is available!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="wouldYouLikeToDownloadLabel">
       <property name="text">
        <string>%1 %2 is now available - you have %3. Would you like to download it now?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="skipThisVersionButton">
         <property name="text">
          <string>Skip This Version</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="remindMeLaterButton">
         <property name="text">
          <string>Remind Me Later</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="installUpdateButton">
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Get Update</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../vmisc/share/resources/icon.qrc"/>
 </resources>
 <connections/>
</ui>
